tests/js/spec/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -212,7 +212,7 @@ describe('WidgetBuilder', function () {
           unit: null,
         },
```

Comment:
should we also update the meta body? https://github.com/getsentry/sentry/blob/09dd1f6b2e7b1a839ad0eac8a038a0f4871e9dec/tests/js/spec/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx#L199